.github/workflows/unit.yaml

```diff
@@ -14,13 +14,13 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, 3.10.8, 3.11.0, pypy-3.6]
+        python-version: [3.7, 3.8, 3.9, '3.10', 3.11, '3.12.0-rc.3', pypy-3.7]
```

Comment:
I think you can just use 3.12 and it should automatically get the latest 3.12 version.

Note that for Twisted, support was removed for 3.7 ... so you can expect test failures with twisted/trunk on 3.7